### DISCUSSION TITLE
FR: Watch alert for leaving phone Bluetooth range

### DISCUSSION THREAD

**Discussion by jamespaulley** [NightscoutFoundation/xDrip#3011] [Ideas]
Could there be an option to have a missed readings alert that will go off on the watch even when the phone is turned off or not in communication with the watch? Might be useful for times when phone battery dies and you haven't noticed, you left your phone elsewhere and not noticed etc?

**Comment by Navid200** [comment]
@jamespaulley Would you please update?
What do you mean exactly?
Do you have a watch that can be a collector and it does not provide alarms?
If the collector is the phone and it is off, how would you expect to get an alarm from the watch considering it is not getting any readings?

**Comment by jamespaulley** [comment]
Hi. So the scenario is this: Your phone is the collector. Without noticing, you leave your phone's bluetooth range. Your phone is not receiving BG data from your Dexcom, nor is it connected to your watch. After 20 minutes, could the xDrip app on the watch trigger an alarm, to alert you to the fact that you have gone out of range?

**Comment by Navid200** [comment]
I suggest you edit the title because right now it may be mistaken with the existing missed reading alert of xDrip, which is something completely different.

You can change the title to for example:
FR: Watch alert for leaving phone Bluetooth range
The title of an issue is very important.  Some people may not even open the issue if they get the wrong impression from the title.

**Comment by Navid200** [comment]
Unfortunately, we don't have active developers supporting WearOS.  My hope is that someone will create a companion app to xDrip to address WearOS.
I'm going to convert this to a discussion, where the conversation can continue.
